engine/util.go

```diff
@@ -33,3 +34,11 @@ func encodeToLegacyFormat(t *Task) ([]byte, error) {
 	// }
 	return json.Marshal(t)
 }
+
+func getenvDefault(varname, value string) string {
```

Comment:
I was following the style of the Go standard library:

https://golang.org/pkg/os/#Getenv